configure

```diff
@@ -723,7 +723,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```

Comment:
It is unclear from the commit message why any changes are being made to this `runstatedir`. Are these changes (and related ones below) needed for the mac build to work?

If these are unrelated changes I request that they be moved to a separate PR.

Reply:
I did not not notice these. It was not intentional.
Maybe this was a difference from running autoconf on Linux vs Mac. I will check.

Reply:
The Linux autoconf generates those lines. The Mac does not.
Both configure scripts seems to run fine on both platforms.
But to minimize unnecessary changes, I will merge the Linux generated configure.